### PR TITLE
operations, network: Declare network hotplug feature as GA

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:

The network hotplug feature has been available since Kubevirt v1.1 as Alpha, followed in v1.3 as Beta.
There have been no changes in the API since and the feedback from users has been positive.

Therefore, the feature is declared GA towards v1.4.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by orelmisan — APPROVED** [review]
Thank you @EdDev.
Unrelated to this PR - IMO it is unclear from the text that NIC hotplug/unplug is only applicable for secondary NICs.
/hold
Lets wait for https://github.com/kubevirt/kubevirt/pull/13059 to be merged before merging this PR.

**Review by ormergi — APPROVED** [review]
LGTM, thank you

File: docs/network/hotplug_interfaces.md
Comment (EdDev) [inline docs/network/hotplug_interfaces.md:27]: We still support old releases that require this FG. Can you place it under a note so users with those versions will still have a ref?

**Review by orelmisan — APPROVED** [review]

**Review by nirdothan — APPROVED** [review]

**Review by ormergi — APPROVED** [review]
LGTM

**Comment by EdDev** [comment]
Raising 3 LGTMs from members of the sig-network.

/approve

**Pull request merged**
